sources/TileDB.Interop/Methods.cs

```diff
@@ -1492,15 +1496,15 @@ public static unsafe partial class Methods
 
         [DllImport("tiledb", CallingConvention = CallingConvention.Cdecl, ExactSpelling = true)]
         [return: NativeTypeName("int32_t")]
-        public static extern int tiledb_filestore_buffer_import(tiledb_ctx_t* ctx, [NativeTypeName("const char *")] sbyte* filestore_array_uri, void* buf, [NativeTypeName("size_t")] ulong size, tiledb_mime_type_t mime_type);
+        public static extern int tiledb_filestore_buffer_import(tiledb_ctx_t* ctx, [NativeTypeName("const char *")] sbyte* filestore_array_uri, void* buf, [NativeTypeName("size_t")] nuint size, tiledb_mime_type_t mime_type);
 
         [DllImport("tiledb", CallingConvention = CallingConvention.Cdecl, ExactSpelling = true)]
         [return: NativeTypeName("int32_t")]
-        public static extern int tiledb_filestore_buffer_export(tiledb_ctx_t* ctx, [NativeTypeName("const char *")] sbyte* filestore_array_uri, [NativeTypeName("size_t")] ulong offset, void* buf, [NativeTypeName("size_t")] ulong size);
+        public static extern int tiledb_filestore_buffer_export(tiledb_ctx_t* ctx, [NativeTypeName("const char *")] sbyte* filestore_array_uri, [NativeTypeName("size_t")] nuint offset, void* buf, [NativeTypeName("size_t")] nuint size);
```

Comment:
I ran into this `ulong` -> `nuint` conversion also when first regenerating the bindings. I have since been manually changing all of these back to `ulong` every time I regenerate, but I am very much in favor of getting this to a state where we won't have to manually edit these files after regenerating.

When I regenerate there are a lot more of these than what I see here, but I think it's likely due to updates that came with the ClangSharpPInvokeGenerator new 15.0.0 version. For me, all instances of `ulong` are replaced with `nuint`, but I notice for you it's only where the native type is `size_t`. 

We can sync on this during our 1:1 today to find out what's missing, but I think it's just that we are using different versions of ClangSharp - I will upgrade today and take another look here with the new version. Great work! :+1: